internal/cli/users.go

```diff
@@ -549,76 +549,82 @@ func importUsersCmd(cli *cli) *cobra.Command {
 		Example: `  auth0 users import
   auth0 users import --connection "Username-Password-Authentication"
+  auth0 users import --connection "Username-Password-Authentication" --users-body "[]"
+  auth0 users import --connection "Username-Password-Authentication" --users-body "$(cat path/to/users.json)"
   auth0 users import -c "Username-Password-Authentication" --template "Basic Example"
-  auth0 users import -c "Username-Password-Authentication" -t "Basic Example" --upsert true
-  auth0 users import -c "Username-Password-Authentication" -t "Basic Example" --upsert true --email-results false`,
+  auth0 users import -c "Username-Password-Authentication" --users-body "$(cat path/to/users.json)" --upsert --email-results
+  auth0 users import -c "Username-Password-Authentication" --users-body "$(cat path/to/users.json)" --upsert --email-results --no-input
+  auth0 users import -c "Username-Password-Authentication" -b "$(cat path/to/users.json)" -u -r
+  auth0 users import -c "Username-Password-Authentication" -t "Basic Example" --upsert --email-results
+  auth0 users import -c "Username-Password-Authentication" -t "Basic Example" --upsert=false --email-results=false
+  auth0 users import -c "Username-Password-Authentication" -t "Basic Example" -u=false -r=false`,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			// Select from the available connection types
-			// Users API currently support database connections
-			options, optionsErr := cli.connectionPickerOptions()
-			if optionsErr != nil {
-				return optionsErr
+			// Users API currently only supports database connections.
+			dbConnectionOptions, err := cli.dbConnectionPickerOptions()
+			if err != nil {
+				return err
 			}
 
-			if err := userConnection.Select(cmd, &inputs.Connection, options, nil); err != nil {
+			if err := userConnection.Select(cmd, &inputs.Connection, dbConnectionOptions, nil); err != nil {
 				return err
 			}
 
-			// Get Connection ID
-			conn, connErr := cli.api.Connection.ReadByName(inputs.Connection)
-			if connErr != nil {
-				return fmt.Errorf("Connection does not exist: %w", connErr)
+			connection, err := cli.api.Connection.ReadByName(inputs.Connection)
+			if err != nil {
+				return fmt.Errorf("failed to find connection with name %q: %w", inputs.Connection, err)
 			}
 
-			inputs.ConnectionID = *conn.ID
-
-			// Present user with template options
-			if templateErr := userImportTemplate.Select(cmd, &inputs.Template, userImportOptions.labels(), nil); templateErr != nil {
-				return templateErr
-			}
+			inputs.ConnectionID = connection.GetID()
 
-			editorErr := userImportTemplateBody.OpenEditor(
-				cmd,
-				&inputs.TemplateBody,
-				userImportOptions.getValue(inputs.Template),
-				inputs.Template+".*.json",
-				cli.userImportEditorHint,
-			)
-			if editorErr != nil {
-				return fmt.Errorf("Failed to capture input from the editor: %w", editorErr)
-			}
+			if inputs.UsersBody == "" {
```

Comment:
If we don't pass a body directly, then we're gonna select a template and use the editor.